core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -402,6 +408,8 @@ object ConsumerGroupCommand {
       "for every consumer group. For instance --topic t1" + nl +
       "WARNING: Group deletion only works for old ZK-based consumer groups, and one has to use it carefully to only delete groups that are not active."
     val NewConsumerDoc = "Use new consumer."
+    val CommandConfigDoc = s"Configs to be passed to Admin Client and Consumer." + nl +
```

Comment:
How about "Property file containing configs to be passed to Admin Client and Consumer."?